challenges/challenge-weather-app/assets/script.js

```diff
@@ -0,0 +1,139 @@
+// API URLs and Keys
+const API_WEATHER = "https://api.openweathermap.org/data/2.5/weather";
+const API_UNSPLASH = "https://api.unsplash.com/search/photos";
+const WEATHER_KEY = "2af3ef06b75eef48a72ed617a00dad79"; // Replace with your OpenWeather API Key
+const UNSPLASH_KEY = "LxKeC1occOPs3F2utkGVSjjUSb0Irq6mzvzWuG7vNS0"; // Replace with your Unsplash Access Key
+
+// DOM Elements
+const photoEl = document.getElementById("photo");
+const thumbsEl = document.getElementById("thumbs");
+const conditionsEl = document.getElementById("conditions");
+const creditUserEl = document.getElementById("credit-user");
+const creditPlatformEl = document.getElementById("credit-platform");
+const searchForm = document.getElementById("search");
+const searchInput = document.getElementById("search-tf");
+
+// Default city
+let currentCity = "London";
+
+// Fetch weather data
+async function fetchWeather(city) {
+  const url = `${API_WEATHER}?q=${city}&appid=${WEATHER_KEY}&units=metric`;
+  try {
+    console.log(`Fetching weather data for: ${city}`);
+    const response = await fetch(url);
+    if (!response.ok) throw new Error(`City "${city}" not found`);
```

Comment:
This works, but a lot of people _always_ use `{}`s around the body of if statements - have a read of https://www.blackduck.com/blog/understanding-apple-goto-fail-vulnerability-2.html for an example of why this is useful.

(This applies to several places in your code)

Reply:
Hi @illicitonion.Thank you for the suggestion! I see how adding the curly braces can improve code clarity and help avoid potential issues. I’ll make sure to update my code to follow this practice. I really appreciate your feedback!